WebRoot/templates/prefs/Pages.template

```diff
@@ -1505,17 +1505,14 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
 							<td class='ZOptionsLabel'>&nbsp;</td>
 							<td class='ZOptionsField'><input id='${id}_CAL_SHOW_DECLINED_MEETINGS' tabindex=0 type=checkbox></td>
 						</tr>
-						<tr><td colspan=2></td></tr>
 						<tr>
 							<td class='ZOptionsLabel'><$=ZmMsg.deleteInviteOnReplyLabel$></td>
 							<td class='ZOptionsField'><input id='${id}_DELETE_INVITE_ON_REPLY' tabindex=0 type=checkbox></td>
 						</tr>
-						<tr><td colspan=2></td></tr>
 						<tr>
-							<td class='ZOptionsLabel' id=id='${id}_CAL_INV_FORWARDING_ADDRESS_label'><$=ZmMsg.forwardInvitesToLabel$></td>
-							<td class='ZOptionsField'><$=ZmMsg.forwardInvitesTo$> <input id='${id}_CAL_INV_FORWARDING_ADDRESS' aria-labelledby='${id}_CAL_INV_FORWARDING_ADDRESS' tabindex=0></td>
+							<td class='ZOptionsLabel' id=id='${id}_CAL_INV_FORWARDING_ADDRESS_label'><$=ZmMsg.forwardInvitesTo$></td>
```

Comment:
Corrected here: https://github.com/Zimbra/zm-web-client/pull/113/commits/d960e6b0c134affd05a417e823d0b27757b832ff#diff-953f1692b15ba4b991582d9b98cc3fb4R1513

Thanks for pointing it out.